pkg/sink/codec/common/helper.go

```diff
@@ -384,3 +377,42 @@ func SanitizeTopicName(name string) string {
 	}
 	return sanitizedName
 }
+
+// UnsafeBytesToString create string from byte slice without copying
+func UnsafeBytesToString(b []byte) string {
+	return *(*string)(unsafe.Pointer(&b))
+}
+
+// UnsafeStringToBytes create byte slice from string without copying
+func UnsafeStringToBytes(s string) []byte {
+	return *(*[]byte)(unsafe.Pointer(
+		&struct {
+			string
+			Cap int
+		}{s, len(s)},
+	))
+}
+
+// UnsafeStringToBinary create binary string from string without copying
+func UnsafeStringToBinary(s string) string {
+	var result string
+	bs := UnsafeStringToBytes(escapeBackslash(s))
```

Comment:
This function is obscure, since:
1. There is no need to use UnsafeStringToBytes, the escape logic can be handled directly when iterating the `s`.

2. The correctness of this function depends on the specific encoding rules. Please explain that there is no problem dealing with utf-8 encoded strings, since the high-bit pattern of UTF-8 bytes is as follows:
```log
Single-byte characters: 0xxxxxxx
Double-byte characters: 110xxxxx 10xxxxxx
Triple-byte characters: 1110xxxx 10xxxxxx 10xxxxxx
Quadruple-byte characters: 11110xxx 10xxxxxx 10xxxxxx 10xxxxxx
```